tests/integration/test_install_uri.py

```diff
@@ -72,17 +72,17 @@ def test_ssh_vcs_install(PipenvInstance):
 def test_urls_work(PipenvInstance):
     with PipenvInstance(chdir=True) as p:
         # the library this installs is "django-cms"
-        path = p._pipfile.get_url("django", "3.4.x.zip")
+        url = "https://github.com/lidatong/dataclasses-json/archive/refs/tags/v0.5.7.zip"
```

Comment:
Go with a lighter-weight test and use a real URL (rather than the old pypi server's artifact's URLs feature).